.github/workflows/tests.yml

```diff
@@ -14,28 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     name: linter-check
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: '12.x'
+          node-version: 16
       - name: Install dev dependencies
         run: yarn --dev
       - name: Run style check
         run: yarn lint && yarn prettier-check
   cypress_no_meilisearch:
     runs-on: ubuntu-latest
-    container: cypress/browsers:node12.18.3-chrome87-ff82
+    container: cypress/browsers:node14.19.0-chrome100-ff99-edge
```

Comment:
You are using node 14 here but your actions are running with node 16

>   with:
>          node-version: 16
>          cache: "yarn"

Reply:
I know :( But there isn't a node 16 container for cypress/browsers currently: 
https://hub.docker.com/r/cypress/browsers/tags

What would you recommend me?

Reply:
If you scroll down a bit I think there are some node16 ones. 
I found this list also (happens that I was looking at that yesterday for one of my projects haha) 
https://github.com/cypress-io/cypress-docker-images/tree/master/browsers#cypressbrowsers

Reply:
Oh thanks a lot! I didn't see that! 🔥